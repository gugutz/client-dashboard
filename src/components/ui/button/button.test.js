import React from 'react'
import renderer from 'react-test-renderer'
import Button from './index'

describe('<Button /> component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Button type="submit">Name</Button>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
