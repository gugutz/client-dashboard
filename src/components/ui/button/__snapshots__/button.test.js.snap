// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> component should render correctly 1`] = `
<button
  className="button"
  type="submit"
>
  Name
</button>
`;

exports[`<Button /> component should support the isFullWidth prop 1`] = `
<button
  className="button is-fullwidth"
  type="submit"
>
  Name
</button>
`;

exports[`<Button /> component should support the isMedium prop 1`] = `
<button
  className="button is-medium"
  type="submit"
>
  Name
</button>
`;

exports[`<Button /> component should support the isOutlined prop 1`] = `
<button
  className="button is-outlined"
  type="submit"
>
  Name
</button>
`;

exports[`<Button /> component should support the isRounded prop 1`] = `
<button
  className="button is-rounded"
  type="submit"
>
  Name
</button>
`;

exports[`<Button /> component should support the isSuccess prop 1`] = `
<button
  className="button is-success"
  type="submit"
>
  Name
</button>
`;
